UN-OCHA/unocha-site
# [UNO-625] Select widget

Refs: UNO-625

This adds and sets up the https://www.drupal.org/project/choices module to improve the UX of the select fields (ex:  country).

<img width="948" alt="Screenshot 2023-05-25 at 11 32 32" src="https://github.com/UN-OCHA/unocha-site/assets/696348/b0f8ed5d-b80c-4cc9-ac95-f5301a2e6802">

Note: `- None -` is automatically selected for empty multi select fields. This is not an issue per se but a bit bad UX. There is a drupal ticket for that: https://www.drupal.org/project/choices/issues/3319674.

### Tests

1. Checkout the branch, recreate the image or run `composer update`
2. Clear the cache, import the config
3. Edit/create a story for example and use select themes, countries etc.
4. Check that the selection is correct
